content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -19,11 +21,16 @@ Ensure that you have the following installed locally:
 
 ## Example Application
 
-The following example uses a basic Express application.
+The following example uses a basic [Express](https://expressjs.com/)
+application. If you're not using Express, that's fine — this guide will also
+work with Koa, Nest.JS
+[and more](/ecosystem/registry/?component=instrumentation&language=js)
```

Comment:
This should be updated to match what we agreed to in https://github.com/open-telemetry/opentelemetry.io/pull/2653#discussion_r1184192841.

Reply:
done